packages/babel-plugin-polyfill-corejs2/src/index.ts

```diff
@@ -116,6 +112,7 @@ export default defineProvider<Options>(function(
 
       if (
         resolved.kind !== "global" &&
+        "object" in meta &&
         meta.object &&
```

Comment:
This is something from TS I hate :sob: I guess I'll have to accept it.

Reply:
It makes sense at least! 
JS is just a wonky language where you can do `meta.object` and if it doesn't exist it's just `undefined`. Would throw a `KeyError` in Python.

So I like to think of it as TS "fixing" some JS if that reframing helps with your acceptance 😅